npm/pkg/dataplane/policies/policy.go

```diff
@@ -0,0 +1,77 @@
+package policies
+
+import (
+	"github.com/Azure/azure-container-networking/npm/pkg/dataplane/ipsets"
+	networkingv1 "k8s.io/api/networking/v1"
+)
+
+type NPMNetworkPolicy struct {
+	Name string
+	// PodSelectorIPSets holds all the IPSets generated from Pod Selector
+	PodSelectorIPSets []*ipsets.IPSet
+	// OtherIPSets holds all IPSets generated from policy
+	// except for pod selector IPSets
+	OtherIPSets []*ipsets.IPSet
```

Comment:
I was thinking of other names, but do you think peeripsets is the right word here ? i can use ruleIpsets ? 

SelectorIpSets -- from pod selector
RuleIPSets -- from rule ?

Reply:
By the way, based on your comments in `OtherIPSets`, will you store all pod selectors from rule and target information in `PodSelectorIPSet`? If so, how to differentiate them? Please check below example yaml.

I understood `PodSelectorIPSets` from `target pod` and others are from `egress` and `ingress` which will be peers to `target pod`. So, I suggested `peersIPSets`.  However, I think `RuleIPSets` is more reasonable since networkpolicy uses `NetworkPolicyIngressRule` and `NetworkPolicyEgressRule`.

```yaml
  podSelector: # target pod
    matchLabels:
      role: db
  policyTypes:
  - Ingress
  - Egress
  ingress:  # peers
  - from:
    - ipBlock:
        cidr: 172.17.0.0/16
        except:
        - 172.17.1.0/24
    - namespaceSelector:
        matchLabels:
          project: myproject
    - podSelector: # another podSelector in rules
        matchLabels:
          role: frontend
    ports:
    - protocol: TCP
      port: 6379
  egress:
  - to:
    - ipBlock:
        cidr: 10.0.0.0/24
    ports:
    - protocol: TCP
      port: 5978

```